.vscode/launch.json

```diff
@@ -75,6 +74,13 @@
                 "${workspaceFolder}/lib/**"
             ],
             "preLaunchTask": "build"
+        },
+        {
+            "name": "Attach to Node Functions",
+            "type": "node",
+            "request": "attach",
+            "port": 9229,
```

Comment:
I believe this can be any open port. However, I didn't manually specify this file so I can't say with certainty, it was autogenerated when setting up my debugging experience, so I suspect the chosen port is the default for these kinds of operations :)